plugins/jobs/web_client/views/JobGraphWidget.js

```diff
@@ -11,7 +11,7 @@ import JobsGraphWidgetTemplate from '../templates/jobsGraphWidget.pug';
 import timingHistoryChartConfig from './timingHistoryChartConfig';
 import timeChartConfig from './timeChartConfig';
 
-export default View.extend({
+const JobGraphWidget = View.extend({
```

Comment:
Yes, it's a marker for what other modules should import this name as, so we can consistently import it as the same thing.